src/core/createLayout.js

```diff
@@ -67,34 +61,27 @@ const createLayout = function() {
     }
     const {
         facetsEl,
-        multiLevelFacetEl,
-        rangeFacetEl,
         selectedFacetsEl
     } = this.options.facet;
+    const {
+        spellCheck
+    } = this.options;
     this.options.facet.selectedFacetTemplate.bind(this);
     this.options.facet.multiLevelFacetTemplate.bind(this);
     this.options.facet.facetTemplate.bind(this);
     this.options.facet.facetItemTemplate.bind(this);
+    if(spellCheck.el) {
+        spellCheck.el.innerHTML = ``;
+        spellCheck.el.appendChild(this.spellCheckWrapper);
+    }
+    if(this.options.facet.el) {
+        this.options.facet.el.innerHTML = ``;
+        this.options.facet.el.appendChild(this.facetsWrapper);
```

Comment:
store `this.options.facet.el` in a variable and use it

Reply:
removed.